src/modules/design/diagram-tools-right/diagram-tools-right.html

```diff
@@ -30,6 +30,9 @@
     <a class="button tool" class.bind="distributeElementsEnabled ? '' : 'button--disabled tool--disabled'" title.bind="distributeElementsEnabled ? 'Distribute Elements Horizontally' : 'Select at least three elements to distribute them horizontally'" click.delegate="distributeElementsHorizontally()" disabled.bind="!distributeElementsEnabled">
       <i class="fas fa-arrows-alt-h"></i>
     </a>
+    <a class="button tool" title="Fit diagram to viewport" click.delegate="fitDiagramToViewport()">
```

Comment:
Does the User know, what a viewport is?

```suggestion
    <a class="button tool" title="Fit diagram to window." click.delegate="fitDiagramToViewport()">
```